instructure/canvas-android
# [Student][MBL-16386] Inbox list redesign

Test plan:
- Verify that all the previously working behavior works as intended (Showing Inbox list, paging, filtering by course/group, filtering by scope, opening messages, pull to refresh, writing messages, updating the list after writing a message or changing/deleting an opened message).
- Verify that the new bulk actions are working correctly.
- Verify that all the data shown is correct (check with the previous version).
- Verify that the routing works correctly.
- The teacher Inbox haven't changed in the scope of this ticket, but some files were changed, so a quick smoke test is enough.
- There were also some changes in the network layer architecture, verify that network requests/parsing works correctly.

refs: MBL-16386
affects: Student
release note: Added bulk actions to Inbox

## Checklist
